### PR TITLE
Save pydap2hdf5

### PR DESCRIPTION
**Pull request by leiyangleon**
## Description

Save the OpenDAP/PyDAP-retrieved weather model data (GMAO and MERRA-2) to an HDF5 file

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have added an explanation of what your changes do and why you'd like us to include them.
- [ ] I have written new tests for your core changes, as applicable.
- [ ] I have successfully ran tests with your changes locally.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by leiyangleon** [comment]
@jlmaurer @dbekaert same problem of missing osr module. I think this should be installed when installing gdal. Not sure what happened exactly.

**Review by dbekaert — APPROVED** [review]
@leiyangleon looks good to me.

**Comment by dbekaert** [comment]
fixed the issue of OSR via #176

File: tools/RAiDER/processWM.py [hunk 1]
```diff
@@ -35,6 +35,9 @@ def getWMFilename(weather_model_name, time, outLoc):
         )
     )
 
+    if weather_model_name is 'GMAO' or 'MERRA2':
+        f = f[:-2]+'h5'
+
```
Comment (dbekaert) [inline tools/RAiDER/processWM.py:40]: @leiyangleon i fixed the "osr" issue we have and confirmed it went fine before i merged this one. I believe this change is impacting one of the unit test to fail for ERA5. Can you have a closer look?

**Pull request merged**